src/main/kotlin/com/msg/gauth/domain/image/service/UploadImageService.kt

```diff
@@ -0,0 +1,26 @@
+package com.msg.gauth.domain.image.service
+
+import com.msg.gauth.domain.admin.exception.FileExtensionInvalidException
+import com.msg.gauth.domain.image.presentation.dto.response.UploadImageResDto
+import com.msg.gauth.global.thirdparty.aws.s3.S3Util
+import org.springframework.stereotype.Service
+import org.springframework.web.multipart.MultipartFile
+import java.util.*
+
+@Service
+class UploadImageService(
+    private val s3Util: S3Util
+) {
+    fun execute(image: MultipartFile): UploadImageResDto {
+        val list = listOf("jpg", "jpeg", "png", "gif")
+
+        val extension = image.originalFilename.toString().split(".")[1].lowercase()
+
+        if(!list.any { it == extension })
```

Comment:
```suggestion
        if(list.any { it == extension })
```

any가 한개라도 맞으면 인거니 !은 뺴야할거같아요

Reply:
깜빡했습니다죄송합니다😭

Reply:
앗 이거 아닌거같은데

Reply:
엄엄엄엄엄